modal/cli/environment.py

```diff
@@ -39,10 +40,25 @@ def __rich__(self):
 def list(json: Optional[bool] = False):
     envs = environments.list_environments()
     table_data = []
+
+    env_var_default = environ.get("MODAL_ENVIRONMENT") or None
```

Comment:
Tiny thing but the `or None` is unnecessary as `os.environ.get()` (really any `dict.get` call) will return `None` if the key is not found.